test_unstructured/unitutils.py

```diff
@@ -0,0 +1,38 @@
+"""Utilities that ease unit-testing."""
```

Comment:
name nit on the module name: it does not read naturally, nor is it obvious what the `uni` means.

Reply:
it's "unit-utils", I can punctuate it if you think that's better, like `unit_utils`.

Reply:
I changed this to `unit_utils.py`.